build error

@claude


> github-pages-demo@0.1.0 build
> next build

[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

   ▲ Next.js 16.0.4 (Turbopack)

   Creating an optimized production build ...
[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
 ✓ Compiled successfully in 4.5s
   Running TypeScript ...
Failed to compile.

./app/components/BreakdownTab.tsx:113:27
Type error: 'percent' is possibly 'undefined'.

  111 |             labelLine={false}
  112 |             label={({ name, percent }) =>
> 113 |               `${name} ${(percent * 100).toFixed(0)}%`
      |                           ^
  114 |             }
  115 |             outerRadius={80}
  116 |             fill="#8884d8"
Next.js build worker exited with code: 1 and signal: null
Error: Process completed with exit code 1.

--- a/app/components/BreakdownTab.tsx
+++ b/app/components/BreakdownTab.tsx
@@ -110,7 +110,7 @@ export function BreakdownTab({
             cy="50%"
             labelLine={false}
             label={({ name, percent }) =>
-              `${name} ${(percent * 100).toFixed(0)}%`
+              `${name} ${percent ? (percent * 100).toFixed(0) : '0'}%`
             }
             outerRadius={80}
             fill="#8884d8"

